modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -15,375 +15,407 @@
 package reverseproxy
 
 import (
-	"context"
-	"crypto/tls"
-	"crypto/x509"
-	"encoding/base64"
-	"fmt"
-	weakrand "math/rand"
-	"net"
-	"net/http"
-	"os"
-	"reflect"
-	"strings"
-	"time"
-
-	"github.com/pires/go-proxyproto"
-	"go.uber.org/zap"
-	"golang.org/x/net/http2"
-
-	"github.com/caddyserver/caddy/v2"
-	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
-	"github.com/caddyserver/caddy/v2/modules/caddytls"
+    "context"
+    "crypto/tls"
+    "crypto/x509"
```

Comment:
You changed tabs to spaces. Please don't do that. We use `gofmt` to format the code.

Reply:
Yes, I definitely made a mistake. Will correct and push.

Reply:
Changed and pushed. Sorry about that. I rarely contribute to other projects and I slipped there.